iina/VideoView.swift

```diff
@@ -228,25 +228,25 @@ class VideoView: NSView {
 
   func setICCProfile(_ displayId: UInt32) {
     typealias ProfileData = (uuid: CFUUID, profileUrl: URL?)
-
     guard let uuid = CGDisplayCreateUUIDFromDisplayID(displayId)?.takeRetainedValue() else { return }
+
     var argResult: ProfileData = (uuid, nil)
-    let dataPointer = UnsafeMutablePointer(&argResult)
-
-    ColorSyncIterateDeviceProfiles({ (dict: CFDictionary?, ptr: UnsafeMutableRawPointer?) -> Bool in
-      if let info = dict as? [String: Any], let current = info["DeviceProfileIsCurrent"] as? Int {
-        let deviceID = info["DeviceID"] as! CFUUID
-        let ptr = ptr!.bindMemory(to: ProfileData.self, capacity: 1)
-        let uuid = ptr.pointee.uuid
-
-        if current == 1, deviceID == uuid {
-          let profileURL = info["DeviceProfileURL"] as! URL
-          ptr.pointee.profileUrl = profileURL
-          return false
+    withUnsafeMutablePointer(to: &argResult) { data in
+      ColorSyncIterateDeviceProfiles({ (dict: CFDictionary?, ptr: UnsafeMutableRawPointer?) -> Bool in
+        if let info = dict as? [String: Any], let current = info["DeviceProfileIsCurrent"] as? Int {
+          let deviceID = info["DeviceID"] as! CFUUID
+          let ptr = ptr!.bindMemory(to: ProfileData.self, capacity: 1)
```

Comment:
I think you want [`assumingMemoryBound(to:)`](https://developer.apple.com/documentation/swift/unsaferawpointer/2431721-assumingmemorybound)

Reply:
I didn't change here, so I'd rather not include this change in this pr. Could you please send another pr to resolve this later?